epochStart/metachain/economics.go

```diff
@@ -93,7 +93,7 @@ func (e *economics) ComputeEndOfEpochEconomics(
 	}
 
 	roundsPassedInEpoch := metaBlock.GetRound() - prevEpochStart.GetRound()
```

Comment:
can these 2 be equal? how?

Reply:
there is no way for these 2 to be equal.